src/otx/algo/action_classification/heads/x3d_head.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+
+"""X3D head implementation."""
+from __future__ import annotations
+
+from torch import Tensor, nn
+
+from otx.algo.action_classification.heads.base_head import BaseHead
+from otx.algo.utils.weight_init import normal_init
+
+
+class X3DHead(BaseHead):
```

Comment:
It just a question. It seems that `MoViNetHead` has a `topk` arugment but this class doesn't. Is it ok?

Reply:
Yes, it is just because that X3D doesn't provide the adjustment for topk loss computation.